index.js

```diff
@@ -28,6 +28,23 @@ app.post('/generate', (req, res) => {
   })
 })
 
+app.put('/qrcards/:id', (req, res) => {
+  QRcode.toDataURL(req.body.url, {scale: 25}, (err, url) => {
+    if (err) throw console.error(err)
+    const qrCode = req.body
+    qrCode.qr = url
+    const query = knex('qrcodes')
+      .where('id', '=', req.params.id)
+      .update(qrCode)
+      .returning('*')
```

Comment:
const query = knex('qrcodes')
      .where('id', '=', req.params.id)

      .update(qrCode)

      .returning('*')



Is this what you are suggesting?